modules/external-project-iam/vars.tf

```diff
@@ -21,4 +21,19 @@ variable gcr_project_id {
 variable gcr_project_iam_roles {
   type        = list(string)
   description = "List of IAM Roles to add GCR project"
-}
\ No newline at end of file
+}
+
+variable gke_service_account {
+  type        = string
+  description = "GKE service account email to add IAM roles in tribe project for"
+}
+
+variable gke_tribe_iam_roles {
```

Comment:
These variable names indicate that this module is becoming very specific to a certain use case -- and less generic. Is this a potential warning sign that we we have a hard time using it for other purposes in the future? Perhaps we shouldn't embed langauage like "tribe" and "gcr" and "gke" -- but instead  use more generic names?

Reply:
yeah, I can change these names to something more generic

Reply:
@sclausson these variables are used in resources where you add specific roles to the specific projects for the gke sa. so I don't know how you can name it to use "more generic names" - I can't name them all as `iam_roles`. can you please tell me what exact names you want me to use?

Reply:
I'll remove "tribe" form var name

Reply:
Well what is the intention of "gke_tribe_iam_roles" within a module called "external project iam"?  Perhaps, generically, this should be called "external_project_iam_roles".

Reply:
it's a list of IAM roles that will be added to the tribe project to GKE SA.

@askoriy the name you picked `external-project-iam` we will changed to `external-project-iam-roles`, is that okay?

Reply:
yes,. its fine

Reply:
okay, cool
I can change `external-project-iam` -> `external-project-iam-roles` and `gke_tribe_iam_roles` -> `gke_parent_iam_roles`
any other changes you'd like me to do?

Reply:
@askoriy , @sclausson ?